### DISCUSSION TITLE
How to disable the bracketed prefix of a script's name when printing to terminal in mpv?

### DISCUSSION THREAD

**Discussion by minnesjay** [mpv-player/mpv#14988] [Q&A] [answered]
In mpv if you use a lua script to print to the terminal it will show the script's name in the terminal output. 

This is usually useful, but I am writing a terminal progress bar so the [ script ] prefix in front of every print is getting in the way. 

How to remove it?

**Comment by guidocella** [comment, marked as answer]
`io.write('foo\n')`

**Reply by minnesjay** [reply]
Thank you
